config/config.go

```diff
@@ -1020,6 +1016,8 @@ var removedConfig = map[string]struct{}{
 	"performance.memory-usage-alarm-ratio":   {}, // use tidb_memory_usage_alarm_ratio
 	"plugin.load":                            {}, // use plugin_load
 	"plugin.dir":                             {}, // use plugin_dir
+	"performance.feedback-probability":       {}, // This feature is deprecated
```

Comment:
Small nit: It was previously deprecated (intent to end of life), but as of this PR it is now removed. I recently cleaned this up and renamed `deprecatedConfig` to `removedConfig` to make it clearer.

Otherwise LGTM

Reply:
I think it is fine.